src/routes/api.js

```diff
@@ -43,7 +50,7 @@ router.post('/add-friend',(req,res)=>{
 router.get('/add-friend', (req, res) => {
```

Comment:
what is this being used for? if adding a friend is getting added as a button, do we need an entire page for it?